src/DonationForms/resources/registrars/templates/fields/Phone.tsx

```diff
@@ -1,15 +1,86 @@
-import InputMask from 'react-input-mask';
-
 import {PhoneProps} from '@givewp/forms/propTypes';
+import {useEffect} from 'react';
+import InputMask from 'react-input-mask';
 
-export default function Phone({Label, ErrorMessage, placeholder, description, phoneFormat, inputProps}: PhoneProps) {
+/**
+ * @unreleased
+ */
+export default function Phone({
```

Comment:
Let's make sure these updates do not cause a conflict with form field manager as I believe this is a shared template.

Reply:
Yep! I did a few tests and the template keeps working as before in phone inputs created with FFM as you can see in the attached screenshot. In fact, I think would be a good upgrade to change the FFM phone input to also use the `intl-tel-input` library.

![image](https://github.com/impress-org/givewp/assets/16308864/a7c989de-46bd-49ef-a190-50ad0808a91a)

Reply:
sounds good, thanks for checking!